packages/e2e/package.json

```diff
@@ -41,7 +41,9 @@
   "devDependencies": {
     "@github/axe-github": "^0.5.0",
     "@playwright/test": "^1.34.3",
-    "axe-playwright": "^1.1.12"
+    "axe-playwright": "^1.1.12",
+    "jest": "^29.7.0",
+    "ts-jest": "^29.2.3"
```

Comment:
Do these two need to be added in `e2e` folder?

Reply:
Good spot, thanks b2af2adce54e885dff03949bc4d076428b2e485e